modules/ROOT/pages/java-ee-libraries.adoc

```diff
@@ -0,0 +1,27 @@
+= Java EE Libraries
+
+In the first Mule runtime version that exclusively supports Java 17 (Mule 4.9.0), Java EE libraries will be removed from connectors. Mule 4.6.0 through Mule 4.9.0 will continue to export Java EE libraries as part of the Container API only when Mule runtime runs on Java 8 or Java 11 to ensure backward compatibility. However, if Mule runtime runs on Java 17 or higher, the Java EE libraries will be removed from connectors, so you will have to include the Java EE libraries manually. For connectors that support multiple Java versions, such as Java 11 and Java 17, the Java EE libraries must match the versions included in Mule runtime. 
```

Comment:
what is the source material for this? The wording doesn't look correct:

* `Java EE libraries will be removed from connectors`  -> This is the SDK documentation. This is the documentation about how to make connectors. 
* ` so you will have to include the Java EE libraries manually` -> Add where? in the application? 

It seems like content from application guide and content from how to develop a Java 17 compatible connector are mixed here.

Reply:
I included info in the next paragraph that says you will need to include the Java EE libraries manually in the connector through a BOM file

Reply:
I updated the wording. Let me know what you think. @marianogonzalez